stable/redis/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 3.4.3
+version: 3.4.4
```

Comment:
As @tompizmor suggests, can you increase the `minor` version instead of `patch` version?
```
-version: 3.4.3
+version: 3.5.0
```